### DISCUSSION TITLE
Welcome to Loxone_LoxStatEdit Discussions!

### DISCUSSION THREAD

**Discussion by mr-manuel** [mr-manuel/Loxone_LoxStatEdit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by integsoft-company** [comment]
Hello, I would love to see some kind of a bulk edit of stats. For examle I do need to substract some number to each consecutive value for meter block in Loxone. For examle I can open stats file and it consist lines like this:
| ID | Value |
| 1 | 10 |
| 2 | 20 |
| 3 | 30 | 
| 4 | 40 |

A need to substract 5 from each row, right now this means to do it for each row manually which is just not possible with thousands of rows...
Tomas

**Reply by mr-manuel** [reply]
We are working on something like this and it's in a very early alpha state.
